src/bin/bat/main.rs

```diff
@@ -200,11 +201,18 @@ pub fn list_themes(cfg: &Config, config_dir: &Path, cache_dir: &Path) -> Result<
     let mut stdout = stdout.lock();
 
     if config.colored_output {
```

Comment:
Would be nice to cover the "not colored output" case, but I don't think you need to do that in this PR. This is clearly an improvement on its own in my view.

Reply:
I agree to implement the same feature for the "not colored output" case but I prefer to do it in another PR after this one is merged into `master`.